src/plugins/credential_management/public/components/edit_credential/edit_credential.tsx

```diff
@@ -161,10 +184,37 @@ export class EditCredentialComponent extends React.Component<
           }}
           toastLifeTimeMs={6000}
         />
+        {
+          this.state.isVisible ?
+            <EuiConfirmModal
+              title="Delete credential permanently?"
+              onCancel={() => { this.setState({ isVisible: false }) }}
+              onConfirm={async () => {
+                const { savedObjects } = this.context.services;
+                try {
+                  //await deleteCredentials(savedObjects.client, [this.props.credential]);
```

Comment:
remove unused code.

Reply:
Thanks for your comments. Will send a pr based on your suggestions next time.